<html>
  <head>
    <meta charset="utf-8" />
<meta name="viewport" content="width=device-width, initial-scale=1" />
<title>Feature Selection for Polygenic Risk Scores using Genetic Algorithm and Network Science | Zhēndòng Shā 沙桢栋</title>
<link rel="shortcut icon" href="https://shazhendong.github.io/favicon.ico?v=1686867543401">
<link href="https://cdn.jsdelivr.net/npm/remixicon@2.3.0/fonts/remixicon.css" rel="stylesheet">
<link rel="stylesheet" href="https://shazhendong.github.io/styles/main.css">
<link rel="alternate" type="application/atom+xml" title="Feature Selection for Polygenic Risk Scores using Genetic Algorithm and Network Science | Zhēndòng Shā 沙桢栋 - Atom Feed" href="https://shazhendong.github.io/atom.xml">
<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Droid+Serif:400,700">


<script async src="https://www.googletagmanager.com/gtag/js?id=G-F07NTD43TN"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-F07NTD43TN');
</script>


    <meta name="description" content="
Authors

Zhendong Sha
Dr. Ting Hu
Dr. Yuanzhu Chen

Abstract
Many human diseases can be attributed to genetic variation..." />
    <meta name="keywords" content="Publication,EvolutionaryComputation" />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.10.0/katex.min.css">
    <script src="https://cdn.bootcss.com/highlight.js/9.12.0/highlight.min.js"></script>
  </head>
  <body>
    <div class="main">
      <div class="main-content">
        <div class="site-header">
  <a href="https://shazhendong.github.io">
  <img class="avatar" src="https://shazhendong.github.io/images/avatar.png?v=1686867543401" alt="">
  </a>
  <h1 class="site-title">
    Zhēndòng Shā 沙桢栋
  </h1>
  <p class="site-description">
    The Dao gives birth to one, one to two, two to three, and three to all things. --Laozi 老子
  </p>
  <div class="menu-container">
    
      
        <a href="/" class="menu">
          Home | 主页
        </a>
      
    
      
        <a href="/tag/publication/" class="menu">
          Publications | 论文
        </a>
      
    
      
        <a href="https://shazhendong.github.io/post/about" class="menu">
          About me | 关于我
        </a>
      
    
  </div>
  <div class="social-container">
    
      
        <a href="https://github.com/shazhendong" target="_blank">
          <i class="ri-github-line"></i>
        </a>
      
    
      
        <a href="https://twitter.com/ZhendongS" target="_blank">
          <i class="ri-twitter-line"></i>
        </a>
      
    
      
    
      
    
      
    
  </div>
</div>

        <div class="post-detail">
          <article class="post">
            <h2 class="post-title">
              Feature Selection for Polygenic Risk Scores using Genetic Algorithm and Network Science
            </h2>
            <div class="post-info">
              <span>
                2021-05-28
              </span>
              <span>
                2 min read
              </span>
              
                <a href="https://shazhendong.github.io/tag/publication/" class="post-tag">
                  # Publication
                </a>
              
                <a href="https://shazhendong.github.io/tag/E6hCpjQfF/" class="post-tag">
                  # EvolutionaryComputation
                </a>
              
            </div>
            
            <div class="post-content-wrapper">
              <div class="post-content">
                <!-- more -->
<h2 id="authors">Authors</h2>
<ul>
<li>Zhendong Sha</li>
<li>Dr. Ting Hu</li>
<li>Dr. Yuanzhu Chen</li>
</ul>
<h2 id="abstract">Abstract</h2>
<p>Many human diseases can be attributed to genetic variations in the genome. Scientists have been identifying genetic variants associated with disease risks using population-based data. With this knowledge, an individual’s genetic liability to a disease can be estimated using the polygenic risk score (PRS), calculated based on their genotype proﬁle. However, selecting the most predictive genetic variants is challenged by the high dimensionality of genomics data. Typically, hundreds of thousands of genetic variants are being tested on their association with a disease risk. Moreover, the effect of a genetic variant on a disease risk is often inﬂuenced by other variants. It is their interactions that contribute to a disease risk. In this research, we propose a feature selection method for PRS assessment that is able to search for combinations of genetic variants using a genetic algorithm and network science. Our method provides accurate predictive models for PRS computation, as well as useful insights into the intertwined relationships among a large number of genetic variants.</p>
<p><em>This work was accepted by CEC2021.</em> [<a href="https://doi.org/10.1109/CEC45853.2021.9504993">Link</a>]</p>

              </div>
              <div class="toc-container">
                <ul class="markdownIt-TOC">
<li>
<ul>
<li><a href="#authors">Authors</a></li>
<li><a href="#abstract">Abstract</a></li>
</ul>
</li>
</ul>

              </div>
            </div>
          </article>
        </div>

        

        

        <div class="site-footer">
  Created with Gridea

  <a class="rss" href="https://shazhendong.github.io/atom.xml" target="_blank">
    <i class="ri-rss-line"></i> RSS
  </a>
</div>

      </div>
    </div>

    <script>
      hljs.initHighlightingOnLoad()

      let mainNavLinks = document.querySelectorAll(".markdownIt-TOC a");

      // This should probably be throttled.
      // Especially because it triggers during smooth scrolling.
      // https://lodash.com/docs/4.17.10#throttle
      // You could do like...
      // window.addEventListener("scroll", () => {
      //    _.throttle(doThatStuff, 100);
      // });
      // Only not doing it here to keep this Pen dependency-free.

      window.addEventListener("scroll", event => {
        let fromTop = window.scrollY;

        mainNavLinks.forEach((link, index) => {
          let section = document.getElementById(decodeURI(link.hash).substring(1));
          let nextSection = null
          if (mainNavLinks[index + 1]) {
            nextSection = document.getElementById(decodeURI(mainNavLinks[index + 1].hash).substring(1));
          }
          if (section.offsetTop <= fromTop) {
            if (nextSection) {
              if (nextSection.offsetTop > fromTop) {
                link.classList.add("current");
              } else {
                link.classList.remove("current");    
              }
            } else {
              link.classList.add("current");
            }
          } else {
            link.classList.remove("current");
          }
        });
      });

    </script>
  </body>
</html>
